client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/config/AdminClientConfig.scala

```diff
@@ -0,0 +1,5 @@
+package ch.epfl.bluebrain.nexus.admin.client.config
+
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+
+final case class AdminClientConfig(baseIri: AbsoluteIri)
```

Comment:
missing docs. Is `baseIri ` containing the prefix too? If that's the case it should be mentioned in the docs. Otherwise, we should create it as I mentioned

```scala
final case class AdminClientConfig(prefix: String, publicIri: AbsoluteIri) {
    lazy val prefixIri: AbsoluteIri      = url"$publicIri/$prefix".value
}
```

Reply:
yes, `baseIri` should include the prefix too. I really don't see the point of having two configuration values just to concatenate them togther.

I'll add the docs.